probe-rs/targets/STM32L4_Series.yaml

```diff
@@ -18,21 +20,23 @@ variants:
       end: 0x20008000
     cores:
     - main
-  - !Ram
-    name: SRAM2
-    range:
-      start: 0x20008000
-      end: 0x2000a000
-    cores:
-    - main
   - !Nvm
+    name: Main_Flash
     range:
       start: 0x8000000
       end: 0x8010000
     cores:
     - main
     access:
+      write: false
       boot: true
+  - !Generic
```

Comment:
Where the heck did the 0x2.... mapping go?

Reply:
yeah it went poof.